dotnet-desktop-guide/net/wpf/whats-new/net90.md

```diff
@@ -0,0 +1,175 @@
+
+This can be enabled by adding the following to your App.xaml:
+
+```xml
+<Application.Resources>
+    <ResourceDictionary>
+        <ResourceDictionary.MergedDictionaries>
+            <ResourceDictionary Source="pack://application:,,,/PresentationFramework.Fluent;component/Themes/Fluent.xaml" />
+        </ResourceDictionary.MergedDictionaries>
+    </ResourceDictionary>
+</Application.Resources>
+```
+
+## AccentColors as SystemColors
+
+Since Windows 10, Accent Color became a cornerstone for visual styling, enhancing consistency across apps by aligning with the system theme. The latest introduction of AccentColors and corresponding brushes in SystemColors simplifies this process for WPF developers, offering them direct access to system accent colors and their variations.
+
+This streamlines the workflow, ensuring apps can seamlessly integrate system-defined aesthetics, ultimately improving the user experience without extra coding overhead. The new API offers a more reliable and efficient way to deliver visually cohesive applications.
+
+1. **Colors**: The following System.Windows.Media.Color are being introduced corresponding to the current accent color of the system and its primary, secondary and tertiary variations in both Light and Dark mode.
+
+    ```cs
+    AccentColor
+    AccentColorLight1
+    AccentColorLight2
+    AccentColorLight3
+    AccentColorDark1
+    AccentColorDark2
+    AccentColorDark3
+    ```
+
+2. **ResourceKey**: Similary, the following System.Windows.ResourceKey are being introduced.
+
+    ```cs
+    AccentColorKey
+    AccentColorLight1Key
+    AccentColorLight2Key
+    AccentColorLight3Key
+    AccentColorDark1Key
+    AccentColorDark2Key
+    AccentColorDark3Key
+    ```
+
+3. **SolidColorBrush**: Similary, the following System.Windows.Media.SolidColorBrush are being introduced.
+
+    ```cs
+    AccentColorBrush
+    AccentColorLight1Brush
+    AccentColorLight2Brush
+    AccentColorLight3Brush
+    AccentColorDark1Brush
+    AccentColorDark2Brush
+    AccentColorDark3Brush
+    ```
+
+### Usage of the `AccentColor` APIs
+
+Usage of the AccentColor APIs, update the applicable properties as below:
+
+```xml
+<Button Content="Sample WPF Button" Background="{x:Static SystemColors.AccentColorBrush}" />
+```
+
+## Hyphen based ligature support
+
+Based on the WPF community feedback, we have now fixed the issue with Hyphen based ligatures not working in WPF. Summary: A call to LsTxtFmt (Text Formatter) is made when a line is being created. Previously, in LsTxtFmt, glyphing was only performed for regular characters. However, after the fix, glyphing will also be performed for special characters, such as hyphens.

```

Comment:
Instead of going into the details like LsTxtFmt, would it be better to provide an image showing the difference ?